scripts/o3de/ExportScripts/export_source_built_project.py

```diff
@@ -487,3 +477,27 @@ def parse_args(o3de_context: exp.O3DEScriptExportContext):
     except exp.ExportProjectError as err:
         print(err)
         sys.exit(1)
+
+# This code is only run by the 'export-project' O3DE CLI command
+
+if "o3de_context" in globals():
+    import pprint
+    pp = pprint.PrettyPrinter(indent=3)
+    global o3de_context
+    global o3de_logger
+
+    # Resolve the export config
+    try:
+        if o3de_context is None:
+            project_name, project_path = command_utils.resolve_project_name_and_path()
+            export_config = exp.get_export_project_config(project_path=project_name)
```

Comment:
This should be passing the project_path as the path parameter.
```suggestion
            export_config = exp.get_export_project_config(project_path=project_path)
```